### PR TITLE
#4061 Make the image name of Shabondi service editable

### PR DESCRIPTION
**Pull request by vitojeng**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4061)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala [hunk 1]
```diff
@@ -72,7 +72,6 @@ object ShabondiDefinitions {
     .displayName("Image name")
     .documentation("The image name of this Shabondi running with")
     .optional(IMAGE_NAME_DEFAULT)
-    .permission(SettingDef.Permission.READ_ONLY) // TODO: #4061
```
Comment (chia7712) [inline ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:75]: 之前怎麼會鎖起來阿?

Comment (vitojeng) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:75]: 說來慚愧...其實我忘了... XD

Comment (chia7712) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:75]: 補個小測試吧~

Comment (vitojeng) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:75]: OK!

**Comment by vitojeng** [comment]
retry collieIT

**Comment by vitojeng** [comment]
> +1 to merge
> 
> I'm digging in the flaky

Got it. I'll stop the jenkins job.

**Comment by chia7712** [comment]
> Got it. I'll stop the jenkins job.

you have to wait the success of building. Otherwise, you can't merge it :)

**Comment by vitojeng** [comment]
> > Got it. I'll stop the jenkins job.
> 
> you have to wait the success of building. Otherwise, you can't merge it :)

I just stop the job. Will retry to build again.

**Comment by vitojeng** [comment]
retry otherIT

**Pull request merged**